README.md

```diff
@@ -4,45 +4,81 @@
 
 ## Development
 
-1. Make sure you have the latest version of Node.js.
+1. Make sure you have Node.js `v6.5.0`.
 
-  ```shell
-  node --version
-  ```
+    ```shell
+    node --version
+    ```
 
-  There are many ways to install Node.js. See [nodejs.org](https://nodejs.org/en/download/) for more info.
+    There are many ways to install Node.js. See [nodejs.org](https://nodejs.org/en/download/) for more info.
 
 2. Clone this GitHub repository.
 
-  ```shell
-  git clone https://github.com/maidsafe/safe_browser.git
-  ```
-
-  If you don't have Git installed, you can download it from [git-scm.com](https://git-scm.com/downloads).
+    ```shell
+    git clone https://github.com/maidsafe/safe_browser.git
+    ```
+  
+    If you don't have Git installed, you can download it from [git-scm.com](https://git-scm.com/downloads).
 
 3. Install the dependencies.
 
-  ``` shell
-  $ cd safe_browser
-  $ npm install
-  ```
+    ``` shell
+    $ cd safe_browser
+    $ npm install
+    ```
 
-4. Build the SAFE Authenticator plugin as explained [here](https://github.com/maidsafe/beaker-plugin-safe-authenticator).
+4. Build the SAFE Authenticator plugin.
 
+    ```
+    $ npm run pack-authenticator
```

Comment:
I would add a comment here stating that it takes several minutes to complete to warn the user preventing him from aborting the process. Alternatively, enhance the script to show some progress bar/messages.